_posts/lisetted/2017-05-22-lisette-turtle_hack.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author: lisetted
+title: "Lisette's Turtle Exercise"
+---
+
+Here's the program I'm embedding:
+<iframe src="https://trinket.io/embed/python/8eb5179d7f" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+
+I didn't recieve Elliot's email about creating a turtle hack before class, so I just copied a turtle (Turtle in space, https://trinket.io/library/trinkets/3bdab3639b) that was on the front page of trinket. After skimming the comments (not really the code:() to understand what the turtle was doing, I decided that I wanted to have the space ship draw as it moved. On my first attempt I tried adding the code for the line towards the beginning after the def statements.
```

Comment:
In addition to my email, this assignment was posted on the Assignments page.  Let me know if there's any confusion.